jedi_vim.py

```diff
@@ -262,10 +263,13 @@ def goto(mode="goto", no_output=False):
                                  for _ in range(4)])
                         f.write('{0}\t{1}\t{2}'.format(tagname, d.module_path,
                             'call cursor({0}, {1})'.format(d.line, d.column + 1)))
-                        f.seek(0)
```

Comment:
`flush` is necessary for the file to be readable/filled by Vim.
The above SO answer might be only relevant for the same Python process maybe (a comment states that `flush` does not work in this case)?!